File: Java/LV1/자연수 뒤집어 배열로 만들기.java
class Solution {
    
    public int[] solution(long n) {
        
        String s = Long.toString(n);
        
        int[] answer = {};
        answer = new int[s.length()];
        for(int i=0;i<s.length();i++){
            answer[i] = s.charAt(s.length()-1-i)-'0';
        }
        
        return answer;
    }
}

File: Lv.1/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(long long n) {
    
    string s=to_string(n);
    vector<int> answer;
    for(int i=0;i<s.length();i++)
    {
        answer.push_back(s[i]-48);
    }
    
    reverse(answer.begin(),answer.end());
    
    return answer;
}
